src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+                FileWriter writer = new FileWriter(toFileName)) {
```

Comment:
Refer to the [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#dont-open-nested-connection-to-files) regarding nested connections